core/modules/image/tests/image.test

```diff
@@ -1937,7 +1940,7 @@ class ImageThemeFunctionWebTestCase extends BackdropWebTestCase {
       'fragment' => $fragment,
     );
     $rendered_element = render($element);
-    $expected_result = '<a href="#' . $fragment . '"><img src="' . $url . '" alt="" /></a>';
+    $expected_result = '<a href="#' . $fragment . '"><img src="' . $url . '" alt="" ' . $lazy_load . '/></a>';
```

Comment:
Just a suggestion for test enhancement, ignore if you disagree... :wink: 

There's currently no test yet for the config setting of lazy_load_images. In one of those tests you could config_set() the value to FALSE and then check if the attribute is *not* there. Just to have a check for whether the setting is respected.

Reply:
I agree, and I think that this has already been suggested by @hosef. The test should not be added here though, as this series of tests is specific to testing image styles. I'll add a separate test to explicitly test whether the attribute is set or not, depending on the config setting.